froide_exam/models.py

```diff
@@ -170,3 +173,11 @@ def get_years(self):
         if self.end_year is not None:
             return list(range(self.start_year.year, self.end_year.year + 1))
         return [self.start_year.year]
+
+
+class PrivateCopy(models.Model):
+    token = models.UUIDField(default=uuid.uuid4)
```

Comment:
Ist es by-design das wir uns die E-Mail-Adresse nicht merken? Erfüllt das dann schon die Anforderung an die "Privatkopie-Weitergabe" (beschränkter Personenkreis von bekannten Leuten osä?)
Das ist jetzt vermutlich nicht so schnell relevant, aber wenn das mal viele werden, braucht token einen Datenbank-Index.

Reply:
Kläre das mal mit Phillip!

Reply:
Ist ohne E-Mail Speicherung in Ordnung. Token ist jetzt auch geindext.